inc/blocks/namespace.php

```diff
@@ -691,3 +700,99 @@ function get_views( string $block_id, $args = [] ) {
 
 	return $data;
 }
+
+/**
+ * Extract inner blocks from attributes.
+ *
+ * @param array $block The block configuration.
+ * @return array
+ */
+function ssr_inner_blocks_shim( array $block ) : array {
+	if ( $block['blockName'] !== 'altis/shim' ) {
+		return $block;
+	}
+
+	// Populate inner blocks by parsing content.
+	$block['innerBlocks'] = parse_blocks( $block['attrs']['content'] ?? [] );
+
+	// Populate inner content with an array of null placeholders so inner blocks are processed.
+	$block['innerContent'] = array_fill( 0, count( $block['innerBlocks'] ), null );
+
+	return $block;
+}
+
+/**
+ * Registers an inner blocks SSR shim.
+ *
+ * @return void
+ */
+function register_ssr_inner_blocks_shim() {
+	register_block_type( 'altis/shim', [
+		'attributes' => [
+			'content' => [
+				'type' => 'string',
+			],
+		],
+		'render_callback' => function ( array $attributes, ?string $inner_content ) : string {
+			return sprintf( '<div>%s</div>', $inner_content );
+		},
+	] );
+}
+
+/**
+ * Register conversion goal validation check.
+ *
+ * @return void
```

Comment:
This return value is incorrect. If the post type doesn't support Gutenberg, we return early with a `bool`. Otherwise we don't have a return. Technically, I guess that would make this `@return bool|void`?

Reply:
That’s the return value of the closure in the array_filter function rather than this function